src/map/pc.cpp

```diff
@@ -2762,7 +2762,7 @@ static void pc_bonus_addeff(std::vector<s_addeffect> &effect, enum sc_type sc, s
 
 	for (auto &it : effect) {
 		if (it.sc == sc && it.flag == flag) {
-			it.rate = cap_value(it.rate + rate, -10000, 10000);
```

Comment:
I think I will need to test this later. Looks like we are capping the sum of the bonus rather than the individual bonus. But even beforehand when I had a combined bonus of >10000 I didn't get a warning in the console, so maybe I'm wrong.

Reply:
The initial bonus is capped just a few lines below when the item bonus is initially created. This line is only specific to stacking the same bonus.

Reply:
Would this matter though? As long as we are capping the final result is all that we are concerned about. The other method would just be capping in two places: at the parse and here at the summation.